### ISSUE TITLE
Option for Assemble

### ISSUE THREAD

**Issue by silvenon** [yeoman/generator-webapp_DEPRECATED#268]
[Assemble](http://assemble.io/) is pretty cool and is [non-trivial](http://fettblog.eu/blog/2013/09/02/using-assemble-io-with-yeoman-ios-webapp-gruntfile/) to plug into this generator. It would be nice if the generator had an option to install it.

Has this been discussed before? I might have seen it somewhere, but searching yielded in no results.

**Comment by silvenon** [comment]
I can do it, I just need a :+1: from you guys. This would be a separate option (or a flag).

**Comment by addyosmani** [comment]
I feel like we may end up punting on this to other existing Assemble generators, but what would be involved in adding a option just for reference?

**Comment by silvenon** [comment]
Sorry, I'm not sure I understand the question, could you rephrase it? If you're asking what I would change, it's pretty much what the article I linked to above said.

**Comment by silvenon** [comment]
Also, what do you mean by "punting on this to other existing Assemble generators"?

**Comment by silvenon** [comment]
I'm closing this because Assemble is still pretty clumsy and badly documented, I wouldn't want to use it yet fo realz. Jekyll seems like a better solution, but that's a Ruby dependency you're trying to avoid, so it's best not to use any templating engine, because each person uses a different one anyway. Plus, there's [generator-jekyllrb](https://github.com/robwierzbowski/generator-jekyllrb) for integration with Jekyll.
